content/posts/android-v2.3.0.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Jellyfin Android v2.3.0 - Improved Integrated Player'
+subtitle: 'Enjoy a more polished Jellyfin experience on your phone'
+author: 'Max Rumpf / Maxr1998'
```

Comment:
All current blog posts don't include a username in the author. But I don't see a problem if you want to include it.

```suggestion
author: 'Max Rumpf'
```